code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -51,6 +53,17 @@
 	set_light(2, 3, l_color = SSticker.cultdat ? SSticker.cultdat.construct_glow : LIGHT_COLOR_BLOOD_MAGIC)
 
 /mob/living/simple_animal/hostile/construct/death(gibbed)
+	SSticker.mode.remove_cultist(mind, FALSE)
+
+	held_body.forceMove(get_turf(src))
```

Comment:
There needs to be a nullcheck for held_body here, things like harvesters are runtiming on death (very bad)